src/components/search/dataSearchQueryBuilder.js

```diff
@@ -9,12 +11,13 @@ export const getDataSimpleSearchQuery = (
     const searchClauses = [
         { type: "label", args: { exact: false, label: searchTerm } },
     ];
+    const pathPrefix = userHomeDir ? userHomeDir : communityDataDir;
     return {
         query: {
-            all: userHomeDir
+            all: !isDetailed
                 ? searchClauses.concat({
                       type: "path",
-                      args: { prefix: userHomeDir },
+                      args: { prefix: pathPrefix },
                   })
                 : searchClauses,
         },
```

Comment:
For detailed search page, we don't want restrict results to any particular path.

Reply:
Is this right? I thought that for logged-out users we still wanted to limit to community data only.